events/notifications.go

```diff
@@ -92,6 +96,62 @@ func (t *NotificationEventProperties) FromMap(m map[string]string) {
 	_ = json.Unmarshal(b, &t)
 }
 
+// labelsTemplate is a helper func to generate the template for displaying labels
+func labelsTemplate(field string) string {
+	return fmt.Sprintf("{{if %s}}**Labels**:\n{{range $k, $v := %s}}`{{$k}}={{$v}}`\n{{end}}{{end}}", field, field)
```

Comment:
```suggestion
	return fmt.Sprintf("{{if %s}}### Labels\n{{range $k, $v := %s}}`**{{$k}}:** {{$v}}`\n{{end}}{{end}}", field, field)
```

Reply:
@moshloop we either have to do bold or code formatting for the label key